app/javascript/components/footer/footer-component.jsx

```diff
@@ -0,0 +1,207 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+
+import MediaQuery from 'react-responsive';
+import Icon from 'components/ui/icon';
+import Carousel from 'components/ui/carousel';
+import Button from 'components/ui/button';
+import ContactUs from 'components/modals/contact-us';
+import Newsletter from 'components/modals/newsletter';
+import wri from 'assets/logos/wri.svg';
+import arrowIcon from 'assets/icons/arrow-down.svg';
+import facebook from 'assets/icons/social/facebook.svg';
+import googleplus from 'assets/icons/social/google-plus.svg';
+import instagram from 'assets/icons/social/instagram.svg';
+import twitter from 'assets/icons/social/twitter.svg';
+import youtube from 'assets/icons/social/youtube.svg';
+import partners from 'data/partners.json';
+import './footer-styles.scss';
+
+class Footer extends PureComponent {
+  render() {
+    return (
+      <MediaQuery minWidth={850}>
+        {isDesktop => (
+          <div className="footerGfw">
```

Comment:
So whilst we are doing this lets update the code to be inline with our other components. Lets start using the same class structure etc. so rename this to `c-footer` or similar.